examples/fabric/image_classifier/train_torch.py

```diff
@@ -56,7 +56,13 @@ def run(hparams):
     torch.manual_seed(hparams.seed)
 
     use_cuda = torch.cuda.is_available()
-    device = torch.device("cuda" if use_cuda else "cpu")
+    use_mps = torch.backends.mps.is_available()
```

Comment:
@awaelchli does Fabric has `auto`?

Reply:
Yes, that's what happens in train_fabric.py, since auto is the default.

Reply:
So we don't need this, right?

Reply:
This is the raw pytorch example. It is meant to show how the example looks like in raw pytorch, and next to it we have the train_fabric.py example. 

It is nice to show how much boilerplate code can be removed when using Fabric/Trainer, and this is the motivation behind having it here.